### ISSUE TITLE
NOTICE: This project is discontinued

### ISSUE THREAD

**Issue by chaoskagami** [chaoskagami/corbenik#81]
It appears some haven't got the memo due to this only being documented on GBAtemp --

This project has been discontinued. Go use something else if you want bugs to be fixed and to get support in using it. If you choose to use Corbenik despite this notice, you're on your own if something breaks.

**Comment by ghost** [comment]
I am furthering the development, under the GPL 3.0 Permissions.

https://github.com/ToadillerYT2/RavioliCFW

Plans are to merge CFWs into that one. (This is actually going to take a while.. Sigh..)

**Comment by ghost** [comment]
And create new menus. I will make a proof of concept when I can.

**Comment by chaoskagami** [comment]
You *may* want to merge the changes from https://github.com/Arcterus/corbenik as well as the credits fixup from https://github.com/hobbledehoy899/corbenik - and bear in mind that this project has a lot of poorly documented code that may or may not be far more complex than Luma in some ways.

I can't say I approve of the name (too memey for me) but hey, that's not my problem as long as proper attribution that it's derived from corbenik is kept as per GPLv3.

**Comment by chaoskagami** [comment]
Considering your past history, I recommend you think *very* hard about whether forking corbenik is a good idea. It might be better to make a new one from scratch or try to port the loader module into Luma or something. I'm not entirely sure you understand the complexity involved here.

**Comment by ghost** [comment]
Most likely I will just port Luma3DS's modules to Corbenik's level. Unless.. Luma3DS doesn't use ctrulib or whatever you used. But I am mostly sure I will have to create Luma Modules from stratch. You can help if you want Chaoskagami 😄 IT IS your project, after all.

EDIT: On merging the changes, will do! 😄

**Comment by chaoskagami** [comment]
Do I look like someone who wants to help, @ToadYillerYT2? No rudeness intended despite phrasing. I've said multiple times, but I've quit. Advice is free, but once it gets tedious I'll have to refrain from replying. I have other things to do.

This repo (e.g. github.com/chaoskagami/corbenik) is my project. Whatever forks are floating around are not. I'm not responsible for whatever you do with my project. Your fork is derived from my project, it isn't my project. I have no responsibilities.

I *really* don't think you should underestimate the amount of effort here. As for "creating new menus" that shouldn't be anywhere near the top of your priority list unless you're trying to make some bastardized rxTools 2.0. And porting a system module (read: rosalina) is going to be anything but easy. You're going to have to adjust the code to allow inserting new modules as well as port patches to the FIRM to load modules beyond the boundary. And if that wasn't enough, rosalina is steeped in use of Luma-esque configuration, and corbenik doesn't configure itself remotely the same way. Never mind that system modules have certain...quirks. And they're difficult to debug.

**Comment by ghost** [comment]
You're right.. They are two different CFW's after all with completely different libaries. But with all due respect, If I were to dual-boot FIRMs by installing the Luma3DS CTRNAND then having Corbenik on the sd card, I guess that's one way to start. But I would have to somehow tell both CFW's to intersect. Any ideas?

**Comment by ghost** [comment]
And to reply as to why I did that name, I was planning to add some stuff from PastaCFW. That is kinda why I named it RavoliCFW. I thought of the whole thing in the shower, water helps me think. 😉

**Comment by ghost** [comment]
Not spamming here, but I just got Kali Linux and a bunch of other Linuxes (dunno if that is a word lol) multibooted with Windows. Kali Linux is derived from Debian Linux. That should help a TON in developing this 😄

**Comment by sykhro** [comment]
Not only it doesn't make any logical sense (Kali derived from Debian aiding in development), but it also makes no practical sense because Kali is the worst possible distro to do development work on.

**Comment by ghost** [comment]
Good point.. Its mostly for exploitation. What I meant was is that it has great github compatibility in its terminal. Will probably take your advice though, and use Debian instead 😉

**Comment by ghost** [comment]
BTW Just out of curiousness Once I make changes do I just compile it like always and it just builds Corbenik with the changes? 

EDIT: I have read the build tutorial, its just that I am not sure I understand.

EDIT2: Is there any way to compile this online?

EDIT3: Does this have a new link? https://j.chaos.moe/job/Corbenik/

**Comment by chaoskagami** [comment]
>And to reply as to why I did that name, I was planning to add some
>stuff from PastaCFW. That is kinda why I named it RavoliCFW. I thought
>of the whole thing in the shower, water helps me think. 😉 

Okay, now I've confirmed you have no clue what you're doing. PastaCFW is ancient history and corbenik already includes all relevant patches from it.

**Comment by chaoskagami** [comment]
>Not spamming here, but I just got Kali Linux and a bunch of other
>Linuxes (dunno if that is a word lol) multibooted with Windows. Kali
>Linux is derived from Debian Linux. That should help a TON in
>developing this 😄 

No, no it will not. Kali is for pentesting, not development. You also seem to not remotely understand linux.

**Comment by chaoskagami** [comment]
I'm going to strongly recommend you stop before you make a fool of yourself. You need to sit down, relax, pick up a book or three on C, and perhaps also Linux and system design while you're at it.

As it is, I do not believe you have the experience neccesary to either understand or properly implement code.

**Comment by chaoskagami** [comment]
Aaaaand, locking discussion. If you want help with your "fork" take it elsewhere.
